kubescape.yaml

```diff
@@ -20,15 +20,14 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 5fd7096d675ca1dc52251081f50686026903f3ec
+      expected-commit: 0e846b2dc56e1c02439da27e74aa238d69b12014
       recurse-submodules: "true"
       repository: https://github.com/kubescape/kubescape
       tag: v${{package.version}}
 
   - uses: go/bump
     with:
       deps: github.com/mholt/archiver/v3@v3.5.1
-      replaces: github.com/mholt/archiver/v3=github.com/anchore/archiver/v3@v3.5.2
```

Comment:
doesn't this removal, reintroduce a previously mitigated CVE?

🔎 Scanning "/tmp/artifacts-1/packages/x86_64/kubescape-3.0.17-r0.apk"
└── 📄 /usr/bin/kubescape
        📦 github.com/mholt/archiver/v3 v3.5.1 (go-module)
            Medium CVE-20[24](https://github.com/wolfi-dev/os/actions/runs/10880442802/job/30187587480?pr=27792#step:9:25)-0406 GHSA-rhh4-rh7c-7r5v